projects/ngrx.io/content/guide/store/actions.md

```diff
@@ -85,6 +85,59 @@ The returned action has very specific context about where the action came from a
 
 </div>
 
+## Signal Actions
+
+There is also the option to dispatch a Signal of type `Signal<Action>`:
```

Comment:
There are no Signal Actions anymore. This docs section needs to be updated to match the new implementation.